app/models/custom_field.rb

```diff
@@ -32,9 +32,8 @@ class CustomField < ActiveRecord::Base
 
   has_many :custom_values, dependent: :delete_all
   acts_as_list scope: 'type = \'#{self.class}\''
-  translates :name,
-             :default_value,
-             :possible_values
+  translates :name, fallbacks_for_empty_translations: false
```

Comment:
This seems to work although I don't understand why. When I read this it speaks to me as if fallbacks would no longer work for `name`. However when trying this, I still get the name entered for `en` even if I have "Deutsch" configured to be my language. Weird bug works.